universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java

```diff
@@ -36,38 +35,47 @@ public ApplicantLayout(ViewUtils viewUtils, ProfileUtils profileUtils) {
     this.profileUtils = checkNotNull(profileUtils);
   }
 
-  protected Content render(Http.Request request, Messages messages, DomContent... mainDomContents) {
-    return render(profileUtils.currentUserProfile(request), messages, mainDomContents);
+  @Override
+  public HtmlBundle getBundle() {
+    return getBundle(new HtmlBundle());
   }
-  /** Renders mainDomContents within the main tag, in the context of the applicant layout. */
-  protected Content render(
-      Optional<UatProfile> profile, Messages messages, DomContent... mainDomContents) {
-    return htmlContent(
-        headContent("Applicant layout title"),
-        body()
-            .with(renderNavBar(profile, messages))
-            .with(mainDomContents)
-            .with(viewUtils.makeLocalJsTag("main"))
-            .withClasses(ApplicantStyles.BODY_BG_COLOR));
+
+  @Override
+  public HtmlBundle getBundle(HtmlBundle bundle) {
+    return super.getBundle(bundle);
+  }
```

Comment:
(blocking) I don't think we need to override these methods since they're identical to those of `BaseHtmlLayout`